addons/mass_mailing/models/mailing.py

```diff
@@ -168,7 +168,7 @@ def _compute_statistics(self):
                 COUNT(CASE WHEN s.bounced is not null THEN 1 ELSE null END) AS bounced,
                 COUNT(CASE WHEN s.exception is not null THEN 1 ELSE null END) AS failed
             FROM
-                mailing_trace s
+                mail_message_res_partner_needaction_rel s
```

Comment:
After discussing with TDE, we think it might be a good idea to rename this to simply 'mail_notification'.

This would have to be done in a separate commit priori to this one, and don't forget to mention that in the "migration" section of the related task.

Reply:
Ok good!
it is a good idea to change also all name is SQL query (mail_notification AS stat -> mail_notification AS notif) or I don't change that ?

Reply:
You can keep it without aliases for now I think, so we reduce the DIFF.